src/upload.ts

```diff
@@ -97,13 +97,24 @@ export function convertBundle(
     bundleId,
     timestamp,
     blockNumber: Number(bundle.blockNumber),
-    transactions: bundle.txs.map((tx) => decodeTx(tx)),
+    transactions: bundle.txs.map((tx) =>
+      decodeTx(tx, bundle.revertingTxHashes)
+    ),
     referrer,
   };
 }
 
-function decodeTx(tx: string): DuneBundleTransaction {
+function decodeTx(
+  tx: string,
+  revertingTxHashes?: Array<string>
+): DuneBundleTransaction {
   const parsed = ethers.utils.parseTransaction(tx);
+  const mayRevert =
+    revertingTxHashes !== undefined
+      ? revertingTxHashes
+          .map((h) => h.toLocaleLowerCase())
+          .includes(parsed.hash.toLowerCase())
```

Comment:
Just want to point out one is `toLocaleLowerCase` and the other is `toLowerCase` -- not sure if this was intended (or autocompleted) as I have never heard of `toLocaleLowerCase` before now.